src/core_modules/capture-core/components/DataEntries/SingleEventRegistrationEntry/SingleEventRegistrationEntry.component.js

```diff
@@ -1,11 +1,17 @@
 // @flow
 import React from 'react';
+import { useSelector } from 'react-redux';
 import { NewEventDataEntryWrapper } from './DataEntryWrapper/NewEventDataEntryWrapper.container';
 import { NewRelationshipWrapper } from './NewRelationshipWrapper/NewEventNewRelationshipWrapper.container';
 import { SelectionsNoAccess } from './SelectionsNoAccess/dataEntrySelectionsNoAccess.container';
 import type { Props } from './SingleEventRegistrationEntry.types';
 
-export const SingleEventRegistrationEntryComponent = ({ showAddRelationship, eventAccess }: Props) => {
+export const SingleEventRegistrationEntryComponent = ({ showAddRelationship, eventAccess, id }: Props) => {
+    const itemId = useSelector((state: ReduxState) => state.dataEntries[id]?.itemId);
+    if (itemId && itemId !== 'newEvent') {
```

Comment:
Can the const [itemId](https://github.com/dhis2/capture-app/blob/DHIS2-17519-app-crashing-when-opening-new-event-from-view-event/src/core_modules/capture-core/components/DataEntries/SingleEventRegistrationEntry/DataEntryWrapper/DataEntry/helpers/constants.js#L5) be used instead?